dockerfiles/test/Dockerfile

```diff
@@ -20,20 +25,21 @@ ENV PATH $SERVICE_HOME/google-cloud-sdk/bin:$PATH
 USER root
 
 # image specific
-RUN apk add --no-cache \
+RUN apt-get update && apt-get install -y --no-install-recommends \
   udev \
   xvfb \
   chromium
-# mongodb not not available in alpine apk repositories post 3.9, switch back to old repo and install from there
-# TODO check if this is still necessary in the future 
-RUN echo 'http://dl-cdn.alpinelinux.org/alpine/v3.9/main' >> /etc/apk/repositories && \
-  echo 'http://dl-cdn.alpinelinux.org/alpine/v3.9/community' >> /etc/apk/repositories && \
-  apk update && \
-  apk add mongodb yaml-cpp=0.6.2-r2
+
+RUN wget -qO - https://www.mongodb.org/static/pgp/server-4.4.asc | apt-key add -
+RUN echo "deb http://repo.mongodb.org/apt/debian buster/mongodb-org/4.4 main" | tee /etc/apt/sources.list.d/mongodb-org-4.4.list
+RUN apt-get update && apt-get install -y mongodb-org
 
 # create default mongodb data directory and give the $USER access
 RUN mkdir -p /data/db && chown -R ${SERVICE_USER} /data/db
 
+RUN npm install --save-dev cypress
+RUN $(npm bin)/cypress verify
```

Comment:
Do we need cypress globally installed?

Reply:
Can double check, this might be a leftover from docker image testing